package.json

```diff
@@ -138,6 +138,7 @@
     "danger": "^6.0.5",
     "del": "6.0.0",
     "enhanced-resolve": "5.8.2",
+    "enquirer": "2.3.6",
```

Comment:
explicitly added dev dependency (used by `@nrwl/tao`, but not re-exported as part of public API)